user_manual/files/access_webgui.rst

```diff
@@ -249,6 +249,21 @@ return and they will receive an email notification, containing the permalink.
 
 .. NOTE:: This is only available if your administrator has enabled email
    notifications
+   
+Share a Private Link
+~~~~~~~~~~~~~~~~~~~~
+
+Another way to share a file or folder with someone is via a private link. 
+It’s a handy way of creating a permanent link for yourself or to make accessing a file or folder, within a share, more efficient. 
+To access the private link, in the Sharing Panel for a file or folder), next to its name you’ll see a small link icon (1), as in the screenshot below. 
```

Comment:
Ah, I know what that was. I'd put one of the number annotations there and didn't remove it when I moved it. Will do that now.